homeassistant/components/climate/broadlink.py

```diff
@@ -0,0 +1,414 @@
+import logging
+import json
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from socket import timeout
+from homeassistant.components.climate import ClimateDevice, \
+    SUPPORT_TARGET_TEMPERATURE,  PLATFORM_SCHEMA, \
+    SUPPORT_OPERATION_MODE, STATE_AUTO
+from homeassistant.const import TEMP_CELSIUS, ATTR_TEMPERATURE, CONF_HOST,\
+    CONF_MAC, CONF_NAME, STATE_ON, STATE_OFF
+
+"""
+Support for Chinese wifi thermostats (Floureon, Beok, Beca Energy)
+configuration.yaml
+climate:
+  - platform: broadlink
+    name: xxx
+    mac: xxxx
+    host: xxxx
+"""
+
+_LOGGER = logging.getLogger(__name__)
+DOMAIN = 'broadlink'
```

Comment:
This is incorrect. The domain of this platform is `'climate'`. Just remove this.